package.json

```diff
@@ -4,10 +4,10 @@
   "description": "",
   "scripts": {
     "start": "DEBUG=ERROR-*,WARN-*,INFO-* node build/bot",
-    "dev": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-*,-DEBUG-helper*,-DEBUG-util* nodemon  -r dotenv/config src/bot.ts",
-    "dev:debug": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-* nodemon src/bot.ts",
-    "dev:info": "DEBUG=ERROR-*,WARN-*,INFO-* nodemon src/bot.ts",
-    "sandbox": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-*,-DEBUG-helper*,-DEBUG-util* nodemon -r dotenv/config",
+    "dev": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-*,-DEBUG-helper*,-DEBUG-util* nodemon -r dotenv/config  -r tsconfig-paths/register src/bot.ts",
+    "dev:debug": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-* nodemon src/bot.ts -r dotenv/config src/bot.ts -r tsconfig-paths/register",
+    "dev:info": "DEBUG=ERROR-*,WARN-*,INFO-* nodemon src/bot.ts -r dotenv/config src/bot.ts -r tsconfig-paths/register",
```

Comment:
What is `-r tsconfig-paths/register` for? We don't have path aliases

Reply:
It's from another PR, it doesn't work without it:
![image](https://user-images.githubusercontent.com/2352112/69809688-86cfb480-11ea-11ea-9876-c4aeea23dc21.png)

We don't have `path` but it seams it makes ts-node to understand better the `baseUrl`.
I removed this in dex-js, and we can re-review it we need to enabled that again there after the refactor